docs/configuration.md

```diff
@@ -942,6 +942,210 @@ editing the `conf` file in a text editor. Use the examples as reference.
     </tr>
 </table>
 
+### dd_configuration_option
+
+<table>
+    <tr>
+        <td>Description</td>
+        <td colspan="2">
+            @warning{Windows only!}
+            Perform additional configuration for the display device:
+            <ul>
+                <li>`disabled` - perform no additional configuration (disables all `dd_` configuration options).</li>
+                <li>`verify_only` - verify that display is active only (required for changing display mode and other options).</li>
+                <li>`ensure_active` - activate the display if it's currently inactive.</li>
+                <li>`ensure_primary` - activate the display if it's currently inactive and make it primary.</li>
+                <li>`ensure_only_display` - activate the display if it's currently inactive and disable all others.</li>
+            </ul>
```

Comment:
Please add the "Choices" row as seen in some of the tables, to display the available options.

![image](https://github.com/user-attachments/assets/ca14994a-f852-412a-ab28-47e62285fcab)

> [!NOTE]
> Applies everywhere a list of choices was added to the description.